### ISSUE TITLE
Support for unreleased Django 1.10, to prevent raising of RemovedInDjango110Warning in Django 1.9

### ISSUE THREAD

**Issue by kiahosseini** [ojii/django-classy-tags#28]
```
/usr/local/lib/python3.4/dist-packages/django/template/loader.py:97: RemovedInDjango110Warning: render() must be called with a dict, not a RequestContext.
  return template.render(context, request)
```

**Comment by ojii** [comment]
Just to clear things up, this is not a 1.9 support issue. 1.9 works fine, this is a warning about support for the unreleased 1.10.

**Comment by kiahosseini** [comment]
Yes, you are right. I noticed the warning error in 1.9.

**Comment by ojii** [comment]
fixed in 0.7.1

**Comment by kiahosseini** [comment]
Thanks for your time and consideration.
